js/renovation/ui/editors/check_box.tsx

```diff
@@ -192,6 +197,19 @@ export class CheckBox extends JSXComponent(CheckBoxProps) {
     return undefined;
   }
 
+  get iconStyles(): { [key: string]: string | number } {
+    const { iconWidth, iconHeight } = this.props;
+
+    const fontSizeMultiplayer = 0.73;
+    const computedIconWidth = typeof iconWidth === 'function' ? +iconWidth() : +iconWidth;
```

Comment:
`iconWidth` can be a string. Does this code work with 'auto', '100%' and another?

Reply:
this const used only for calculating fontSize, and yes, it doesn't work with such strings. and font-size (tick size) will not change.
trying to come up with something

Reply:
done via effect updateIconFontSize